browser/ui/sidebar/sidebar_controller.cc

```diff
@@ -260,4 +273,14 @@ void SidebarController::SetSidebar(Sidebar* sidebar) {
       profile_, ServiceAccessType::EXPLICIT_ACCESS));
 }
 
+SidebarWebPanelController* SidebarController::GetWebPanelController() {
+  CHECK(IsWebPanelFeatureEnabled());
```

Comment:
How about having this check inside the SidebarWebPanelController's constructor instead of this place?